toml.abnf

```diff
@@ -117,14 +117,19 @@ underscore = %x5F                  ; _
 
 ;; Float
 
-float = integer ( frac / ( frac exp ) / exp )
+float =  integer ( frac / ( frac exp ) / exp )
```

Comment:
maybe now would be a good time to change this to?

```abnf
float = integer ( exp / frac [ exp ] )
```

Reply:
Seems reasonable.